src/contrib/mlir/utils.hpp

```diff
@@ -26,6 +26,87 @@ namespace ngraph
     {
         namespace ngmlir
         {
+            enum class OpType
```

Comment:
These are not general mlir utils and should be part of the cpu backend. You can create a new file there : `callback_utils.hpp` or the like.